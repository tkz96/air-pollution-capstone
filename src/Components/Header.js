import React from 'react';

const Header = () => (
  <header>
    <h1>Air Quality Report</h1>
  </header>
);

export default Header;
