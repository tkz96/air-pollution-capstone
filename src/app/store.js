import { configureStore } from '@reduxjs/toolkit';
import countrySlice from './features/countrySlice';

const store = configureStore({
  reducer: {
    country: countrySlice,
  },
});

export default store;
