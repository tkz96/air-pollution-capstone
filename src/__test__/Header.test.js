import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import Header from '../Components/Header';
import store from '../app/store';

describe('header tests', () => {
  it('render header', () => {
    const tree = render(
      <Provider store={store}>
        <Router>
          <Header />
        </Router>
      </Provider>,
    );

    expect(tree).toMatchSnapshot();
  });
});
