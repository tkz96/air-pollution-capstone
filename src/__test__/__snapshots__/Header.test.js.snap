// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header tests render header 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header>
        <div
          class="logo"
        >
          <h3>
            Air Quality Report
          </h3>
        </div>
        <div
          class="micandgear"
        >
          <img
            alt="mic"
            class="headerImages"
            src="mic.png"
          />
          <img
            alt="gear"
            class="headerImages"
            src="gear.png"
          />
        </div>
      </header>
    </div>
  </body>,
  "container": <div>
    <header>
      <div
        class="logo"
      >
        <h3>
          Air Quality Report
        </h3>
      </div>
      <div
        class="micandgear"
      >
        <img
          alt="mic"
          class="headerImages"
          src="mic.png"
        />
        <img
          alt="gear"
          class="headerImages"
          src="gear.png"
        />
      </div>
    </header>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
